src/app/component/comments/components/add-comment/add-comment.component.html

```diff
@@ -1,28 +1,42 @@
-<form class="main-wrapper"
-      [formGroup]="addCommentForm"
-      [ngClass]="!commentId ? 'wrapper-comment': 'wrapper-reply'"
-      (ngSubmit)="onSubmit()">
+<form
+  class="main-wrapper"
+  [formGroup]="addCommentForm"
+  [ngClass]="!commentId ? 'wrapper-comment' : 'wrapper-reply'"
+  (ngSubmit)="onSubmit()"
+>
   <app-user-profile-image
     [imgPath]="avatarImage"
     [firstName]="firstName"
-    [additionalImgClass]="'comments'">
+    [additionalImgClass]="'comments'"
+  >
   </app-user-profile-image>
   <div class="input-submit" *ngIf="!commentId; else reply">
-    <textarea placeholder="{{ 'homepage.eco-news.comment.placeholder.add-a-comment' | translate }}"
-              formControlName="content"></textarea>
-    <button class="primary-global-button"
-            [disabled]="!addCommentForm.valid">
-      {{ "homepage.eco-news.comment.comment"| translate }}
+    <textarea
+      placeholder="{{
+        'homepage.eco-news.comment.placeholder.add-a-comment' | translate
+      }}"
+      formControlName="content"
+    ></textarea>
+    <button
+      class="primary-global-button"
+      style="width: 505px"
+      [disabled]="!addCommentForm.valid"
+    >
+      {{ "homepage.eco-news.comment.comment" | translate }}
     </button>
   </div>
   <ng-template #reply>
     <div class="input-submit">
-      <textarea placeholder="{{ 'homepage.eco-news.comment.placeholder.add-a-reply' | translate }}"
-                formControlName="content"
-                [maxlength]="replyMaxLength"></textarea>
-      <button class="primary-global-button"
-              [disabled]="!addCommentForm.valid">
-        {{ "homepage.eco-news.comment.reply"| translate }}
+      <textarea
+        placeholder="{{
+          'homepage.eco-news.comment.placeholder.add-a-reply' | translate
+        }}"
+        formControlName="content"
+        [maxlength]="replyMaxLength"
+        style="467px"
```

Comment:
style haven't property. are you mean:  style="width: 467px" ??

Reply:
✅ Typo has been fixed